helm/postgres/values.yaml

```diff
@@ -155,16 +155,15 @@ postgresVersion: 14
 #   name: bootstrap-sql
 #   key: bootstrap.sql
 
-# standby sets whether or not to run this as a standby cluster. Both of the
-# values below are required to enable a standby cluster. Setting "enabled" to
+# standby sets whether or not to run this as a standby cluster. Setting "enabled" to
 # "true" eunables the standby cluster while "repoName" points to a pgBackRest
 # archive to replay WAL files from, and "host" and "port" point to a primary
 # cluster from which to stream data.
 # standby:
 #   enabled: false
 #   repoName: repo1
-#   host: "<primary-ip>"
-#   port: "<primary-port>"
+#   host: "192.0.2.2"
+#   port: "5432"
```

Comment:
Thank you for the update, one last small tweak before merging would be greatly appreciated. Could we change this port value to an integer? This will make it easier for users during configuration as this is defined as an integer in our [spec](https://github.com/CrunchyData/postgres-operator/blob/master/pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go#L554-L557).

Relatedly, I opened a [PR](https://github.com/CrunchyData/postgres-operator/pull/3582) to better reflect this nuance in our documentation.